BraveUI/Popover/PopoverController.swift

```diff
@@ -417,7 +417,7 @@ extension PopoverController {
 
     containerView.transform = CGAffineTransform(translationX: translationDelta.x, y: translationDelta.y)
       .scaledBy(x: scale, y: scale)
-      .rotated(by: rotation)
+//            .rotated(by: rotation)
```

Comment:
I think we could add a flag to enable/disable rotation for the `PopoverController`

Reply:
we've had a bunch of problems with rotation here, I think it may be better to just fully remove it. Will bring it up with rest of iOS team